playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -29,7 +29,7 @@
 
 - name: Backup etcd
   include: ./etcd/backup.yml
-  when: openshift_upgrade_skip_etcd_backup | default(false) | bool
+  when: not openshift_upgrade_skip_etcd_backup | default(false) | bool
```

Comment:
It's only recently been added and was never documented. I'd be in favor of making it easier to read. I'll include that in my work to include etcd upgrades by default.